app/views/pages/new_project.html.erb

```diff
@@ -0,0 +1,172 @@
+<!--
+  This example requires Tailwind CSS v2.0+ 
+  
+  This example requires some changes to your config:
+  
+  ```
+  // tailwind.config.js
+  module.exports = {
+    // ...
+    plugins: [
+      // ...
+      require('@tailwindcss/forms'),
+    ]
+  }
+  ```
+-->
+<h1 class="py-6 my-6 text-center"><%= t('pages-new_project-title') %></h1> <!-- New wedding booklet project -->
```

Comment:
The string translation comments are fine but also probably unnecessary.

There are usually IDE plugins that can help with this kind of thing, like https://marketplace.visualstudio.com/items?itemName=shanehofstetter.rails-i18n

Reply:
I tried installing this plugin, but it doesn't seem to be working for me, I'm not sure what's missing to get it working.